tools/packaging/kata-deploy/local-build/kata-deploy-build-and-upload-payload.sh

```diff
@@ -15,19 +15,33 @@ cp ${KATA_DEPLOY_ARTIFACT} ${KATA_DEPLOY_DIR}
 
 pushd ${KATA_DEPLOY_DIR}
 
-IMAGE_TAG="${REGISTRY}:kata-containers-$(git rev-parse HEAD)"
+IMAGE_TAG="${REGISTRY}:kata-containers-$(git rev-parse HEAD)-$(uname -m)"
 
 echo "Building the image"
-docker build --tag ${IMAGE_TAG} .
+if [ "$(uname -m)" = "s390x" ]; then
+	docker build \
+		--build-arg IMG_NAME=clefos \
```

Comment:
Follow-up discussion: clefos 7 is quite old (but above Docker file is based on centos 7 as well). See my comment above.